dashboard/app/views/projects/projects.html.haml

```diff
@@ -3,7 +3,11 @@
 .projects-table-wrapper.center
   = render partial: 'projects/links'
 
-  %div.loading{'ng-show' => 'projects.length === 0'}
+  %div{'ng-show' => 'projects.length === 0 && projectsLoaded'}
+    %div{style: "padding: 10px;"}
```

Comment:
hound is on the fritz, so you can ignore this as long as its passing your local commit hooks and circleci